concourse/client/model.py

```diff
@@ -221,6 +266,24 @@ def find_tid(p):
                             return task_id
         return find_tid(plan)
 
+    def contains_version_ref(self, resource_version_ref: str):
```

Comment:
nit: A comment might be helpful for future maintainers

Reply:
done